oculus_ros2/CMakeLists.txt

```diff
@@ -7,44 +7,112 @@ find_package(oculus_interfaces REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rclpy REQUIRED)
 find_package(std_msgs REQUIRED)
+find_package(sensor_msgs REQUIRED)
+# find_package(rqt_reconfigure REQUIRED)
 find_package(rcl_interfaces REQUIRED)
 
-find_package(oculus_driver QUIET)
-if(NOT TARGET oculus_driver)
-    include(FetchContent)
-    FetchContent_Declare(oculus_driver
-        GIT_REPOSITORY   https://github.com/ENSTABretagneRobotics/oculus_driver.git
-        GIT_TAG          v1.0
-    )
-    FetchContent_MakeAvailable(oculus_driver)
-endif()
+find_package(image_transport REQUIRED)
+find_package(cv_bridge REQUIRED)
+find_package(OpenCV REQUIRED)
```

Comment:
Better to specify a min version for `OpenCV`. This package is supposed to work with `Humble`, so you can have its OpenCV version as your minimum.

Reply:
Where can I found the openCV minmial version? I do not found it on the documentation but I think `find_package(OpenCV 4.2.0 REQUIRED)` will be good.

Reply:
[ROS2 humble](https://www.ros.org/reps/rep-2000.html#humble-hawksbill-may-2022-may-2027) is shipped with `OpenCV 4.5.4`, so you can have that version as your minimum but if it works with `4.2.0` then it should be good.